portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -53,6 +57,23 @@ public static String getParameter(HttpServletRequest request, String name, Strin
     return value;
   }
 
+  /**
+   * Attempts to parse an long string. Returns the default value if the string is null.
```

Comment:
The term "long string" could be misinterpreted. Without context I interpret "long string" as a string with a lot of characters, not as the string representation of a long.
```suggestion
   * Attempts to convert a string into a long. Returns the default value if the string is null.
```